Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.22-alpine AS builder
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+
+RUN go mod download
+RUN go mod verify
+
+COPY ./ ./
+
+RUN go build -ldflags="-w -s" -o server ./cmd
+
+FROM scratch
+COPY --from=builder app/server /usr/server
```

Comment:
off note that if your command ever would need to make outbound tls (https) connections you would need to copy the CA bundle too